README.md

```diff
@@ -504,14 +504,144 @@ Validates list input.
 
 # Template & Static files
 
+API Star is particularly designed around building APIs, but is equally
+capable of being used to build web applications.
+
 ## Templates
 
+To include templates in your application, create a `templates` directory
+alongside your `app.py` file, and use `app.render_template(template_name, **context)`.
+
+By default `jinja2` is used for template rendering.
+
+**templates/index.html**:
+
+```
+<html>
+    <body>
+        {% if name %}
+        <h1>Hello, {{ name }}!</h1>
+        {% else %}
+        <h1>Hello!</h1>
+        {% endif %}
+    </body>
+</html>
+```
+
+**app.py**:
+
+```
+from apistar import App, Route
+
+
+def welcome(app: App, name=None):
+    return app.render_template('index.html', name=name)
```

Comment:
Is there a possibility to render same handler using different renderers specified in route or accept header?

For example:


```python
def welcome(name: str=None) -> dict:
    return {'name': name}

routes = [
    Route('/', method='GET', handler=welcome, renderer=template('index.html')),
    Route('/', method='GET', handler=welcome, renderer='json'),
    Route('/', method='GET', handler=welcome, renderer='yaml'),
]
```

Or renderer could be chosen by accept header:

```python
routes = [
    Route('/', method='GET', handler=welcome, renderer={
        'text/html': template('index.html'),
        'application/json': 'json',
        'application/x-yaml': 'yaml',
    }),
]
```